https://discourse.onlinedegree.iitm.ac.in/t/164277

I have fixed it thank you .</p><hr>

<h4>Jivraj (#11)</h4>
<p>Hi Aishik,</p>
<p>Pls add context to your query, without that we won’t be able to understand, where exactly you are facing problem.</p>
<aside class="quote group-ds-students" data-post="10" data-topic="164277" data-username="23f2005325">
<div class="title">
<div class="quote-controls"></div>
 23f2005325:</div>
<blockquote>
<pre><code class="lang-auto">openai.AuthenticationError: Error code: 401 - {'error': {'message': 'Your authentication token is not from a valid issuer.', 'type': 'invalid_request_error', 'param': None, 'code': 'invalid_issuer'}}
</code></pre>
</blockquote>
</aside>
<p>Possible reasons for this issue:</p>
<ol>
<li>Not using anand sir’s proxy url for sending requests.</li>
<li>Token not being correct.</li>
</ol><hr>

<h4>23f2005325 (#10)</h4>
<p>getting this issue :</p>
<pre><code class="lang-auto">openai.AuthenticationError: Error code: 401 - {'error': {'message': 'Your authentication token is not from a valid issuer.', 'type': 'invalid_request_error', 'param': None, 'code': 'invalid_issuer'}}
</code></pre><hr>

<h4>Jivraj (#8)</h4>
<p>Hi Guddu ,</p>
<p>I didn’t make any changes to file and it worked for me.